### DISCUSSION TITLE
Self hosted NextCache config

### DISCUSSION THREAD

**Discussion by toddgeist** [vercel/next.js#47852] [Help] [answered]
### Summary

In the announcement  of Next Cache on this page
https://nextjs.org/blog/next-13-2#nextjs-cache-when-self-hosted

There is a reference to configuring how the cache works when self-hosted and that it can be configured or disabled. But there is no link to docs on how to do it. I have found a few keys in NextConfig.experimental that might be involved, but that is it.

Is there more information somewhere?

Thanks :-)

**Comment by kmanimurugan** [comment]
I am also facing the same issue.
@Nextjs support please help us. How to enable cache header in Self hosted sites.

**Comment by leerob** [comment, marked as answer]
https://nextjs.org/docs/app/api-reference/next-config-js/incrementalCacheHandlerPath
